### PR TITLE
pmu updates

### PR DESCRIPTION
**Pull request by atishp04**

### CODE REVIEW

File: src/ext-pmu.adoc [hunk 1]
```diff
@@ -602,6 +602,73 @@ The possible error codes returned in `sbiret.error` are shown in
...
+get event specific information for multiple events at one shot by writing an
+entry for each event in the shared memory. Each entry in the shared memory
+must be encoded as follows:
+
+[#table_event_info_format]
+.Event info entry format
+[cols="1,2,7", width=90%, align="center", options="header"]
+|===
+| Word | Name        | Encoding
+| 0    | event_idx   | BIT[0-19] - Describes `event_idx`
```
Comment (lftan) [inline src/ext-pmu.adoc:627]: Suggest changing "BIT[0-19]" to "BIT[0:19]", using the same bit format representation as other sections.
The same applies to the reset of BIT[] below.

File: src/ext-pmu.adoc [hunk 2]
```diff
@@ -602,6 +602,73 @@ The possible error codes returned in `sbiret.error` are shown in
...
+                       BIT[20-31] - Reserved for the future purpose. Must be
+                       zero.
+| 1    | output      | BIT[0] - Boolean value to indicate `event_idx` is
+                       is supported or not. The SBI implmenentation must update
+                       this field if valid `event_idx` and `event_data`
+                       (if applicable) specified in this entry.
+                       BIT[1-31] - Reserved for future purpose. Must be zero.
+| 2-3  | event_data  | BIT[0-63] - Valid only if `event_idx.type` is either
```
[the remark this answers is not in the document]
Comment (atishp04) [reply src/ext-pmu.adoc:635]: Done. PTAL.

File: src/ext-pmu.adoc (hunk 1 above)
Comment (atishp04) [reply src/ext-pmu.adoc:627]: Done. PTAL.

**Comment by atishp04** [comment]
Closing the PR as it was just intended to trigger the CI.

**Pull request closed without merging**